packages/router-devtools/src/index.tsx

```diff
@@ -1,2 +1,5 @@
-export { TanStackRouterDevtools } from './TanStackRouterDevtools'
-export { TanStackRouterDevtoolsPanel } from './TanStackRouterDevtoolsPanel'
+console.warn(
+  '[@tanstack/router-devtools] This package has moved to @tanstack/react-router-devtools. Please switch to the new package at your earliest convenience, as this package will be dropped in the next major version release.',
+)
+
+export { TanStackRouterDevtoolsInProd as TanStackRouterDevtools } from '@tanstack/react-router-devtools'
```

Comment:
There should be separate Panel export as well yes?

Reply:
Not too sure, I think it was something about portalling it or something 🤷🏼‍♂️.

Just don't want to break anything for anyone who upgrades.

Reply:
Just checked the source, it's to do with portalling the devtools into a shadowDOM target.

Related issues:
- https://github.com/TanStack/router/issues/1065
- https://github.com/TanStack/router/issues/1246
- https://github.com/TanStack/router/issues/1247

Reply:
Did a bit more investigation into the Panel component.

It's essentially, the devtools component itself, without the floating component aspect to it. So the panel component won't have resize functionality or the concept of being anchored to a position ("bottom-right" | "top-left" | "...").

The idea is to be able to portal/attach the devtools component onto a ShadowDOM node.

Reply:
Hmm, I think I need an example of something that's _not_ working.

I've added examples for solid/react of non-nested usage (/examples/framework/basic-non-nested-devtools), where the devtools are outside of the RouterProvided and are passed the router.

Reply:
To the best of my efforts have a I added a Panel to the core, and added wrappers in the framework packages, and a forward from `@tanstack/router-devtools`

I'd just haven't been able to confirm it works as expected.

Reply:
Is it maybe better to "soft-launch" this, so that the existing `@tanstack/router-devtools` stay untouch apart from a message to migrate over, so we can see how that goes instead of nuking the working implementation in `@tanstack/router-devtools` on day 1? It would allow anyone hitting an issue to stay on `@tanstack/router-devtools`

Reply:
Examples of injecting the panel into shadowdom can be found at router/examples/(solid|react)/basic-devtools-panel